src/store/modules/ConnectModule.js

```diff
@@ -25,8 +26,8 @@ class ConnectModule extends VuexModule {
   }
 
   @Action
-  async setClientId(clientId) {
-    LocalStorage.save(CLIENT_ID_KEY, clientId);
+  async setClientId(clientId = '') {
```

Comment:
Is it possible to remove default value? IMO we should use that only in cases when `clientId` exists